app/src/main/java/com/bongtu/baekseo/data/mapper/KakaoMapMapper.kt

```diff
@@ -0,0 +1,15 @@
+package com.bongtu.baekseo.data.mapper
+
+import com.bongtu.baekseo.data.dto.map.KakaoMapResponse
+import com.bongtu.baekseo.data.model.map.Place
+
+fun KakaoMapResponse.PlaceDocument.toModel(): Place {
+    return Place(
+        id = id,
+        name = placeName,
+        address = addressName,
+        roadAddress = roadAddress,
+        latitude = y.toDoubleOrNull() ?: 0.0,
+        longitude = x.toDoubleOrNull() ?: 0.0,
+    )
+}
```

Comment:
p2: 이렇게 간소화해서 사용도 가능할 것 같습니다!
```suggestion
fun KakaoMapResponse.PlaceDocument.toModel() = Place(
    id = id,
    name = placeName,
    address = addressName,
    roadAddress = roadAddress,
    latitude = y.toDoubleOrZero(),
    longitude = x.toDoubleOrZero(),
)
```

Reply:
좋네요~ 이대로 수정할게요ㅎㅎ